src/playground/workers/prettierWorker.ts

```diff
@@ -109,7 +116,14 @@ async function formatWithPrettier(
 			tabWidth: options.indentWidth,
 			printWidth: options.lineWidth,
 			filepath: options.filepath,
-			plugins: [parserBabel, pluginEstree, pluginCss, pluginGraphql],
+			plugins: [
+				parserBabel,
+				pluginEstree,
+				pluginCss,
```

Comment:
nit: sorting alphabetically is better for readability maybe?
```suggestion
				pluginCss,
				pluginEstree,
```